### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#641] [Galaxy Trivia] [answered]
🟦◻️🟥✝️
🟩🟧🟥🦁
☀️◽🟦🟥

**Comment by peterbaccaro** [comment, marked as answer]
1. ICELAND
2. SRI LANKA
3. TAIWAN

![flags](https://user-images.githubusercontent.com/74666079/235166915-cb28dd51-8d77-4f3a-83e8-2f8ba7ba5698.png)

**Reply by ParanoidUser** [reply]
Hot! The last one ◽☀️🟦🟥 != ☀️◽🟦🟥

**Reply by peterbaccaro** [reply]
3. PHILIPINES

![flag](https://user-images.githubusercontent.com/74666079/235302552-ad1efd96-9d81-4933-8ef3-e2bd4f510a30.png)
